backend/apps/ai/common/create_chunks_and_embeddings.py

```
@@ -0,0 +1,56 @@
+    Creates chunks and embeddings from given texts using OpenAI embeddings.
+
+    Args:
+        all_chunk_texts (list[str]): List of text chunks to embed.
+        content_object: The object to associate the chunks with.
+        openai_client: Initialized OpenAI client instance.
+
+    Returns:
+        list[Chunk]: List of Chunk instances (not saved).
+    """
+    try:
+        last_request_time = datetime.now(UTC) - timedelta(seconds=DEFAULT_LAST_REQUEST_OFFSET_SECONDS)
```

Ruff (0.11.9)

Line too long (102 > 99)

(E501)